pepper/Dockerfile

```diff
@@ -4,7 +4,7 @@ MAINTAINER Felix Leupold <felix@gnosis.pm>
 ENV PEPPER /home/docker/pequin/
 
 USER root
-RUN apt-get install -y inotify-tools libgtest-dev time vim tmux
+RUN apt-get install -y inotify-tools libgtest-dev time tmux
```

Comment:
Can we find a solution for this while keeping vim installed? Also not really related to this PR so could be its own commit.